src/minimal-live-connect.js

```diff
@@ -26,10 +28,12 @@ import { CallHandler } from './handlers/call-handler'
 function _minimalInitialization (liveConnectConfig, externalStorageHandler, externalCallHandler) {
   try {
     const callHandler = CallHandler(externalCallHandler)
-    const storageHandler = StorageHandler(liveConnectConfig.storageStrategy, externalStorageHandler)
+    const storageStrategy = liveConnectConfig.gdprApplies ? StorageStrategy.disabled : liveConnectConfig.storageStrategy
+    const storageHandler = StorageHandler(storageStrategy, externalStorageHandler)
```

Comment:
We probably need to set the new parameters explicitly here when `gdprApplies` is true